src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/FormattingLanguageServerClient.cs

```diff
@@ -105,6 +115,48 @@ private RazorDocumentRangeFormattingResponse Format(RazorDocumentRangeFormatting
                 var generatedHtml = codeDocument.GetHtmlDocument().GeneratedHtml;
                 generatedHtml = generatedHtml.Replace("\r", "", StringComparison.Ordinal).Replace("\n", "\r\n", StringComparison.Ordinal);
 
+                var editHandlerAssembly = Assembly.Load("Microsoft.WebTools.Languages.LanguageServer.Server, Version=16.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
```

Comment:
hahahahahah, now this is 💪 worthy!

Real question though. Do you envision us keeping the reflection logic or maybe working with WEbTools to get IVT access?

Reply:
We'll do one of two things:

* Continue to use reflection
* Modify the tests to spin up the actual HTML LSP server instance, and communicate with it for tests